client/src/components/Navbar/index.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '@mui/material/Link';
```

Comment:
There's no need to use the the `react-router-dom' Link inside of `material-ui` Link
```suggestion
import { Link } from 'react-router-dom';
```